ofl/joan/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,5 @@
+<p>Joan is a serif typeface suitable for books and magazines. It supports the complete Latin Plus character set.</p>
```

Comment:
"Latin Plus" doesn't mean much for user, I opened an issue for the designer to improve the description: https://github.com/PaoloBiagini/Joan/issues/8